src/main/java/gov/cabinetofice/gapuserservice/service/DepartmentService.java

```diff
@@ -20,4 +22,16 @@ public List<DepartmentDto> getAllDepartments() {
                 .map(departmentMapper::departmentToDepartmentDto)
                 .collect(Collectors.toList());
     }
+
+    public Department getDepartmentById(int id) {
+        return departmentRepository.findById(id).orElseThrow(() -> new DepartmentNotFoundException("Department not found"));
+    }
+
+    public Department updateDepartment(int id, String departmentName, String ggisId) throws DepartmentNotFoundException {
+        Department department = departmentRepository.findById(id).orElseThrow(() -> new DepartmentNotFoundException("Department not found"));
```

Comment:
Might pass in the department instead, for a similar reason as above

Reply:
Done that